gsl_programmation/views.py

```diff
@@ -179,3 +180,34 @@ def patch_status_simulation_projet(request, pk):
 
     SimulationProjetService.update_status(simulation_projet, new_status)
     return redirect_to_simulation_projet(request, simulation_projet)
+
+
+def simulation_form(request):
+    if request.method == "POST":
+        form = SimulationForm(request.POST)
+        if form.is_valid():
+            print("CHIRAC")
+            print(request.user)
+            print(form)
+            print(form.cleaned_data)
+            SimulationService.create_simulation(request.user, form.cleaned_data)
+
+            return redirect(
+                "programmation:simulation_list"
+            )  # Replace 'success_url' with your actual success URL
+    else:
+        form = SimulationForm()
+        context = {
+            "breadcrumb_dict": {
+                "links": [
+                    {
+                        "url": reverse("gsl_projet:list"),
+                        "title": "Liste des projets",
+                    },
+                ],
+                "current": "Création d'une simulation de programmation",
+            }
+        }
+        context["form"] = form
+
```

Comment:
Ajoute aussi le `<title>` dans le contexte : 
```suggestion
        context["title"] = "Création d’une simulation de programmation"

```

Reply:
Ah, je viens de comprendre l'intérêt !!